perun-base/src/main/resources/perun-roles.yml

```diff
@@ -6790,116 +6850,160 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   group-addFormItem_ApplicationForm_ApplicationFormItem_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
```

Comment:
yes, it concerns Vo or group, however, do we change them / set something for them or for the given form only? Same applies for update/delete items or forms. Perhaps, we might discuss it to clarify it. What do you say?
@Johaney-s @xflord

Reply:
We discussed this, and we probably want to keep it this way. We might remove it in the future if we do not want it.